### DISCUSSION TITLE
See attached Albums at a glance

### DISCUSSION THREAD

**Discussion by jvanvolken** [photoprism/photoprism#3203] [Ideas]
**_(Background)_** I'm going through a ton of pictures and collecting subsets in different Albums. I cannot do this all in one sitting, so I've been tackling this in many separate sittings. **_(Problem)_** What makes this difficult is I'm unsure what I've added to what Albums already, so I'm having to go back and forth to figure out where I left off. **_(Solution)_** I propose some type of indicator attached to each picture/video indicating which Albums they are attached to. 

The implementation of Albums in Photoprism is already very similar to TAGS in other applications, I believe this should be embraced.

Could look something like this:
![Album Tags Concept](https://user-images.githubusercontent.com/43481443/219372006-d996e7dd-0e02-4936-80ea-12eb3bb27eb3.png)

**Comment by graciousgrey** [comment]
We plan to add this functionality on the details tab of the edit dialog.

**Reply by jvanvolken** [reply]
This wouldn't be very practical - I would have to 'edit' any picture I want to check its location(s). How would this be different than keywords or labels? 

Right now I'm having to 'favorite' the images that belong to a subset in order to see at a glance which ones I've added and then I go back into favorites and move all of the images to their Album.

**Reply by graciousgrey** [reply]
The idea is having an input field for albums on the details tab. This will allow you see to what albums the photo belongs to as well as adding it to additional albums or removing it from albums.

We do also have an open issue to customize the information shown on the card view, however this is very critical performance wise, so definitely not a quick win.

**Reply by jvanvolken** [reply]
Gotcha... Still, having to 'edit' the image to access the details tab just isn't going to work when managing my library unless the details feature is overhauled from its current state. In the edit/details view, I'm unable to see other images for context or even use arrow keys to move between images. You're also unable to select multiple images and mass move to an album from the details tab.

- I'll stick to favoriting pictures and then moving all favorites to an album, then unfavoriting them until it's easier to manage albums.

**Reply by graciousgrey** [reply]
I totally understand your point.

As I said, we probably won't find time for this any time soon but feel free to leave a :+1: in here: https://github.com/photoprism/photoprism/issues/1164
